Programming Foundations: Algorithms/Factorial-test/test_unit/Facttest.rb

```diff
@@ -0,0 +1,21 @@
+require "test/unit"
+require_relative "/home/hema/RoR-Learning-Path/Programming Foundations: Algorithms/Factorial-test/fact"
+
+
+
+class Facttest < Test::Unit::TestCase
+
+  def test_positive
+    assert_equal(6, Fact.new(3).fact)
+  end
+
+  def test_Zero
+    assert_equal(1, Fact.new(0).fact)
+  end
+
+  def test_negative
```

Comment:
Make it more readable
See https://github.com/srijitamukherjee/RoR-Learning-Path/pull/2/files#r331490367